packages/dashboard-frontend/src/services/bootstrap/PreloadData.ts

```diff
@@ -45,8 +45,8 @@ export class PreloadData {
   @lazyInject(CheWorkspaceClient)
   private readonly cheWorkspaceClient: CheWorkspaceClient;
 
-  @lazyInject(DevWorkspaceClient)
-  private readonly devWorkspaceClient: DevWorkspaceClient;
+  @lazyInject(Index)
+  private readonly devWorkspaceClient: Index;
```

Comment:
@olexii4 Could you elaborate why Index is here?

Reply:
Good catch. It was automatically renamed with the file. I have fixed it.